documentation/tutorials/customizing-tokens-via-cli.md

```diff
@@ -0,0 +1,95 @@
+# Customizing tokens via CLI
+
+In this tutorial, we'll dig into more details and go beyond what the dApp allows.
+We'll customize the parameters of synthetic tokens via the command line by creating a BTC token
+margined in ETH and interacting with it via the command line.
+
+## Financial engineering note
+
+The financial engineering of this token is _complicated_, because the creator is exposed to both Bitcoin and ETH price moves.
+
+## Prerequisites
+
+All of our tutorials require that you complete the steps in [Prerequisites](./prerequisites.md). If you are new to the UMA
+system, start with [Creating tokens locally[(./creating-tokens-locally.md) to get a gentle introduction.
+
+Make sure you have testnet ETH or are running locally. All commands should be run from the `core` directory.
+
+## Token creation
+
+First launch truffle console:
+
+```
+$(npm bin)/truffle console --network=<networok>
+```
+
+Pass `--network=test` for local runs. We'll type the rest of the commands in this document into the Truffle console.
```

Comment:
Should we tell them what network to use for rinkeby?

Reply:
Sure, makes sense.